include/tlspool/commands.h

```diff
@@ -15,8 +15,10 @@ extern "C"
 #define _usleep(usec) (Sleep((usec) / 1000))
 #define poll(fds, nfds, timeout) WSAPoll(fds, nfds, timeout)
 #include <winsock2.h>
+#define DPRINTF(...) tlog (TLOG_UNIXSOCK, LOG_DEBUG, __VA_ARGS__)
```

Comment:
Mweh, je gebruikt `TLOG_UNIXSOCK` voor alles... bovendien is dit nou juist spul dat ik niet log.